schema/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package doc defines the main types supported by tables in source plugins
```

Comment:
```suggestion
// Package schema defines types supported by tables in source plugins
```

Or change the package name